galaxy/scripts/values_table.py

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
I think you should move this under a new dir `galaxy-helm/scripts` not `galaxy-helm/galaxy/scripts` since it's a convenience script for us so no reason to have it be packaged as part of the pushed chart. Removing the other scripts unrelated to the chart from there in https://github.com/galaxyproject/galaxy-helm/pull/251